src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/on.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'on(event, handler)'
+excerpt: 'Browser module: page.on method'
+---
+
+Allows the definition of an event handler function that will be executed every time the specified event is emitted.
+
+| Parameter       | Type   | Default | Description                                                                                                                                                                                                                           |
+|-----------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| event        | string  | `''`    |  Event to attach handler to. Currently, only `'console'` event is supported.                                                                                                               |
+| handler        | function  | `null`    |  A function to be called every time the specified event is emitted.                                                                                                               |
+
+
+<Blockquote mod="warning" title="">
```

Comment:
Maybe this could be a note instead of a warning. A warning feels like it's too strong for case.

Reply:
I'm +1 for keeping this as a warning, as without this, user tests will hang 🤔

Reply:
Looking at our style guide in the [writers-toolkit](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#cautions), I'd say this falls somewhere in between a `note` and `warning`.

We don't have a `caution` type like we do in the Grafana docs, but I see we do have an `attention`. Maybe we can use that instead?

Reply:
That looks good! Fixed in [73ec98b](https://github.com/grafana/k6-docs/pull/1308/commits/73ec98bd3085a12424bcd03faf73b8faac8226b3).